Gemfile.lock

```diff
@@ -63,6 +63,7 @@ GEM
     sass-listen (4.0.0)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
+    wdm (0.1.1)
```

Comment:
Please do not make changes to `Gemfile.lock`
Don't do a `git add` for this file